### PR TITLE
feat: add project impact description

### PR DESCRIPTION
**Pull request by flagrede**
## Description

Closes: regen-network/regen-registry#1738

- handle project-specific impact description

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [ ] provided a link to the relevant issue or specification
- [ ] provided instructions on how to test
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### How to test

1. https://deploy-preview-2067--regen-marketplace.netlify.app/project/C01-001

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] verified React components follow DRY principles
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

**Comment by flagrede** [comment]
@erikalogie you can check that we now display on-chain impact descriptions for project here: https://deploy-preview-2067--regen-marketplace.netlify.app/project/C01-001

**Comment by erikalogie** [comment]
@flagrede this looks good!

File: web-marketplace/src/pages/CreditClassDetails/CreditClassDetails.utils.ts [hunk 1]
```diff
@@ -95,16 +99,20 @@ export const parseCreditClassVersion = (
...
 
 const normalizeImpact = ({
   impact,
+  projectImpact,
   sanityImpact,
   sdgs,
 }: NormalizeImpactInput) => ({
   name: impact?.['schema:name'] || sanityImpact?.name,
+  description:
+    projectImpact?.['schema:description'] || sanityImpact?.descriptionRaw,
```
Comment (blushi) [inline web-marketplace/src/pages/CreditClassDetails/CreditClassDetails.utils.ts:115]: @erikalogie if there's some generic description on sanity for an impact, do we still want to display it on the project page as on https://deploy-preview-2067--regen-marketplace.netlify.app/project/C01-002 or on the credit class page https://deploy-preview-2067--regen-marketplace.netlify.app/credit-classes/C01 ?

Comment (blushi) [reply web-marketplace/src/pages/CreditClassDetails/CreditClassDetails.utils.ts:115]: I don't see anything on the latest design: https://www.figma.com/file/seUiQM2SPt78KJiDL0Ea8m/Basic-Ecocredits-Ledger-View-%2F-Marketplace?node-id=9376%3A250796&mode=dev

Comment (erikalogie) [reply web-marketplace/src/pages/CreditClassDetails/CreditClassDetails.utils.ts:115]: No we don't want to display the generic description anymore on the project page, just the specific one if it exists in the metadata.

**Review by blushi — APPROVED** [review]

**Pull request merged**